cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/ReadSegmentRangeCommand.java

```diff
@@ -99,9 +78,9 @@ private File createFileAndDirectory(String fileName) throws IOException {
      * @throws IOException if the file create/write fails.
      * @throws Exception if the request fails.
      */
-    private void readAndWriteSegmentToFile(SegmentHelper segmentHelper, String segmentStoreHost, String fullyQualifiedSegmentName,
+    public void readAndWriteSegmentToFile(SegmentHelper segmentHelper, String segmentStoreHost, String fullyQualifiedSegmentName,
```

Comment:
Why this needs to be public?

Reply:
We are reusing the functionality provided by this method in another place, which I why I needed to make it public